pkg/builder/parameters.go

```diff
@@ -64,6 +64,12 @@ func collectSharedParameters(sp *spec.Swagger) (namesByJSON map[string]string, r
 		if (p.In == "query" || p.In == "path") && p.Name == "name" {
 			return nil // ignore name parameter as they are never shared with the Kind in the description
 		}
+		if p.In == "query" && p.Name == "fieldValidation" {
+			return nil // keep fieldValidation parameter unshared because kubectl uses it (until 1.27) to detect SSA support
```

Comment:
```suggestion
			return nil // keep fieldValidation parameter unshared because kubectl uses it (until 1.27) to detect server-side field validation support
```